src/Symfony/Component/Validator/Constraints/Null.php

```diff
@@ -11,17 +11,19 @@
 
 namespace Symfony\Component\Validator\Constraints;
 
-use Symfony\Component\Validator\Constraint;
-
-/**
+/*
```

Comment:
Hum, that's true. That makes me wonder: how about replacing the class_alias with a proper class declaration?
`class Null extends IsNull {}`
This file will never be loaded under PHP7, so that will not trigger any error!
The good thing is that we'll be able to keep a proper docbloc comment, for the benefit of IDEs like phpStorm.